Pod/Classes/Mocking/MockingView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  MockingView.swift
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 22/02/2018.
+//  Copyright © 2018 Allegro Group. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/// View stubing layout defining properties.
+/// Some of layout properties (e.g. `safeAreaInsets`, or `traitCollection`) are read-only,
+/// so this view can be used as container which helps to force custom layout.
+@objc(AGMockingView)
+public class MockingView: UIView {
```

Comment:
When one could use `MockingView` instead of `DeviceMockingView`? Did you consider making one a subclass of another? It looks like lots of code duplication.

Reply:
Subclass is impossible because of `mockParameters` property has different type in `MockingView` and `DeviceMockingView`.

`MockingView` might be useful for simple testing cases, eg: Developer needs to test some view for larger content size category or custom trait collection.

`DeviceMockingView` should be used for testing view on specific device eg. iPad (so mockParameters would contains whole set of iPad UI specific values like: size and safe area insets) and additionally values which are useful for describing reference images (eg. orientation [on iPad height > width not always means portrait orientation], device name)

Reply:
> Subclass is impossible because of mockParameters property has different type in MockingView and DeviceMockingView.

What about making just one subclass of `UIView` that has two initializers (`ViewMockable` and `DeviceViewMockable`)? This kind of generic class would just be one purpose class that would be passed to `FBSnapshotVerify(...)` method. I think it would make usage a little simpler while preserving all the functionality. What do you think?